packages/wdio-cli/src/interface.js

```diff
@@ -174,55 +178,59 @@ export default class WDIOCLInterface extends EventEmitter {
         }
 
         /**
-         * print stdout and stderr from runners
+         * add empty line between "pending tests" and results
          */
-        if (isFinished) {
-            /* istanbul ignore else */
-            if (this.interface.stdoutBuffer.length) {
-                this.interface.log(chalk.bgYellow.black('Stdout:\n') + this.interface.stdoutBuffer.join(''))
-            }
-            /* istanbul ignore else */
-            if (this.interface.stderrBuffer.length) {
-                this.interface.log(chalk.bgRed.black('Stderr:\n') + this.interface.stderrBuffer.join(''))
-            }
-            /* istanbul ignore else */
-            if (this.messages.worker.error) {
-                this.interface.log(chalk.bgRed.black('Worker Error:\n') + this.messages.worker.error.map(
-                    (e) => e.stack
-                ).join('\n') + '\n')
-            }
+        if (this.jobs.size) {
+            this.interface.log()
         }
 
+        this.printSummary()
+        this.updateClock()
+    }
+
+    printReporters() {
+        this.interface.clearAll()
+
         /**
-         * add empty line between "pending tests" and results
+         * print reporter output
          */
-        if (this.jobs.size) {
+        for (const [reporterName, messages] of Object.entries(this.messages.reporter)) {
+            this.interface.log(chalk.bgYellow.black(`"${reporterName}" Reporter:`))
+            this.interface.log(messages.join(''))
             this.interface.log()
         }
 
-        this.interface.log(
+        /**
+         * print stdout and stderr from runners
+         */
+        /* istanbul ignore else */
```

Comment:
why `/* istanbul ignore else */` if this if has no else?

Reply:
Removed.